### DISCUSSION TITLE
Beta versions

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#162] [Q&A] [answered]
How many people have signed up as beta tester ?  I ask as I just last week did promote the Cruiser app to be very reliable.
Would it not be better to release a beta version first when introducing complex changes like the latest BRouter Cruiser integration ?
(I downgraded to the previous version).

**Comment by devemux86** [comment, marked as answer]
@0709wiwiwi thanks, good question!

Currently around 150, a lot of users, so we can probably try the Beta version for such changes next time.

---

Cruiser (Beta) instructions are here:

- https://github.com/devemux86/cruiser/discussions/36

**Comment by devemux86** [comment]
It's very simple, we can enter and exit the Google Play testing at any time.
Just need to remember to back up favorites & settings in case of uninstall.

See the instructions in the Beta announcement's image:
- https://github.com/devemux86/cruiser/discussions/36

**Reply by devemux86** [reply]
We enabled the flag for _"Whether to show the user a prompt to keep the app's data when the user uninstalls the app"_.
It works on Android 10+. Doesn't it work with downloaded maps, etc.?

- https://github.com/devemux86/cruiser/discussions/80
